packages/eslint-config-airbnb-base/README.md

```diff
@@ -15,8 +15,8 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+. It
 1. Ensure packages are installed with correct version numbers by running:
   ```sh
   (
-    export PKG=eslint-config-airbnb-base;
-    npm info "$PKG" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG"
+    PKG=eslint-config-airbnb-base
+    npm info "$PKG" peerDependencies --json | sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG"
```

Comment:
Yes, and that's why every command that's not `npm` here is prefixed with `command`. If there were others, they all would have `command` as well.

I'm not sure why it's confusing - most will just copy-paste the command and don't need to understand it. Those that do, will understand `command`.